### ISSUE TITLE
Getting useraccounts to work with Angular

### ISSUE THREAD

**Issue by vertangelx** [Urigo/angular-blaze-template#11]
Using the blaze template `{{> atForm}}` from `useraccounts:bootstrap` is giving an error about a missing template: `HTML template does not exists!`

Here is what I have in my Angular view file:

```
<template name="atForm">
    {{> atForm}}
</template>

<blaze-template name="atForm"></blaze-template>
```

Any help greatly appreciated!

**Comment by Urigo** [comment]
Should be fixed when `angular-with-blaze@1.3.1` will be released

**Comment by Urigo** [comment]
I've just released [1.3.1](https://github.com/Urigo/angular-meteor/releases), please re-open if it still doesn't work for you
